tests/mock_tables/config_db.json

```diff
@@ -370,6 +370,16 @@
         "pfc_asym": "off",
         "speed": "40000"
     },
+    "PORT|Ethernet128": {
```

Comment:
I see why you added etp33, can't we use existing ports.

Reply:
don't think so